style.css

```
@@ -0,0 +1,71 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
```

```suggestion
*
,*::before
,*::after{
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}
```
You should usecss reset rule like this...

---

yes i understand we can use multiple selector if they have same property but what if they dont have any similiarity...and here you repeat universal selector multiple times its little bit creating confusion for me can u please explain?. 
*
,*::before
,*::after{

---

yes, it was by mistake, I have updated the suggestion (above code), you should write after and before without a universal selector...